modules/programs/par/internal/editor/editor.go

```
@@ -0,0 +1,209 @@
+	if promptName != "" {
+		// Sanitize prompt name for filename
+		safeName := strings.ReplaceAll(promptName, " ", "-")
+		safeName = strings.ReplaceAll(safeName, "/", "-")
+		fileName = fmt.Sprintf("par-prompt-%s-*.md", safeName)
+	}
+
+	tempFile, err := l.fileSystem.CreateTemp(tempDir, fileName)
+	if err != nil {
+		return "", fmt.Errorf("failed to create temporary file: %w", err)
+	}
+	defer l.fileSystem.Remove(tempFile.Name())
```

golangci-lint (1.64.8)

Error return value of `l.fileSystem.Remove` is not checked

(errcheck)